### PR TITLE
Add long_description for PyPI

### PR DESCRIPTION
**Pull request by ocsw**
**What this PR does / why we need it**:
Adds a long_description for PyPI.

**Which issue(s) this PR fixes**

Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added a long_description for PyPI.
```

### CODE REVIEW

**Comment by ocsw** [comment]
Hm.  Should I just use README instead?

File: setup.py [hunk 1]
```diff
@@ -87,6 +87,7 @@ def install_requires():
     version=find_meta('version'),
     license=find_meta('license'),
     description=find_meta('description'),
+    long_description=read('DESCRIPTION.md'),
```
Comment (econchick) [inline setup.py]: Could this just be the `README.rst`?

Perhaps not for this - if we do it for others - but we could also add the changelog, i.e. `read('README.rst', 'CHANGELOG.rst')`

**Review by econchick — CHANGES_REQUESTED** [review]

**Comment by ocsw** [comment]
https://stackoverflow.com/questions/16367770/my-rst-readme-is-not-formatted-on-pypi-python-org

Seems they do support it, more or less?

**Comment by ocsw** [comment]
No, that's got repo stuff, not package stuff.

**Comment by ocsw** [comment]
Er, didn't see your comment @econchick before I commented and re-uploaded to fix the tests.  Let's discuss.

**Comment by econchick** [comment]
PyPI always has had rst support, IIRC they just added markdown support.

rst is very common to use for descriptions, even if it isn't parsed "prettily." @ocsw people still do, though (eg [ex1](https://github.com/spotify/ramlfications/blob/master/setup.py#L55) ([rendered](https://pypi.org/project/ramlfications/)), [ex2](https://github.com/python-attrs/attrs/blob/master/setup.py#L81-L90) ([rendered](https://pypi.org/project/attrs/)))

File: setup.py (hunk 1 above)
Comment (matt0bi) [reply setup.py]: Just talked IRL about this with @ocsw  - we settled on using `README.rst` for the `long_description` as the simplest, most legible approach.

**Comment by ocsw** [comment]
@econchick changed to your suggestion

**Comment by descent3me** [comment]
LGTM +1

**Review by econchick — APPROVED** [review]

**Pull request merged**